src/core/src/preprocess/preprocess_steps_impl.cpp

```diff
@@ -542,7 +542,7 @@ void PostStepsList::add_convert_impl(const element::Type& type) {
 
 void PostStepsList::add_convert_layout_impl(const Layout& layout) {
     m_actions.emplace_back(
-        [layout](const Output<Node>& node, PostprocessingContext& context) {
+        [&layout](const Output<Node>& node, PostprocessingContext& context) {
```

Comment:
Not sure about this, do we really need to catch by reference here? After return from function, this 'layout' can be destroyed and lambda will have invalid reference inside and will crash when called. Let's wait for CI :-)